### PR TITLE
docs(bil): Billing feature branch MTA-4067

### PR DESCRIPTION
**Pull request by pshambulingappa**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: console/billing/index.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+meta:
+  title: Billing Documentation
+  description: Dive into Scaleway billing with our quickstart guides, how-tos, tutorials and more.
```
Comment (SamyOubouaziz) [inline console/billing/index.mdx:4]:
```suggestion
  description: Dive into Scaleway billing with our quickstart guides, how-tos, and more.
```

File: bare-metal/dedibox/how-to/link-dedibox-account.mdx [hunk 2]
```diff
@@ -59,7 +59,7 @@ You can now manage your current and future Dedibox servers from the Scaleway con
       * If you do not have a SEPA as your preferred payment method in either account, the preferred payment method of credit card remains the same for both Scaleway Cloud and Scaleway Dedibox invoices.
       * If you use different credit cards for Dedibox and Scaleway Cloud, the card registered with Scaleway Cloud becomes the default payment method for both Scaleway Cloud and Scaleway Dedibox invoices.
         <Message type="tip">
-          Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/organization/billing) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
+          Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/billing/payment) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
```
Comment (jpaturel-scw) [inline bare-metal/dedibox/how-to/link-dedibox-account.mdx:62]: replace the here by https://console.scaleway.com/organization/billing/payment

File: console/billing/how-to/add-billing-contact.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The **billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department. 
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/add-billing-contact.mdx:24]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.

File: console/billing/how-to/add-payment-method.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+Before you can order Scaleway resources, you must add your payment method to your account.
+
+<Macro id="iam-requirements" />
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/add-payment-method.mdx:24]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.

File: console/billing/how-to/add-payment-method.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+You can choose between two payment methods: **credit card** or **SEPA mandate.**
+
+### How to add a credit card
+
+1. Enter the details of your credit or debit card.
+2. Click **Add credit card** to confirm. A [3D Secure](https://en.wikipedia.org/wiki/3-D_Secure) validation is initiated. Depending on your bank, you may be asked to enter a code received by SMS or via an application on your smartphone.
+3. Enter the code and validate the form.
+    <Message type="note">
+     When you register a new credit card, an authorization charge of **1€** is debited from your account. This **Authorization Charge** is a dummy transaction made to check the validity of the card and will carry a 4-digit validation code, displayed on your bank account statement. The authorization charge will be reimbursed within 48 to 72 hours.
```
Comment (jpaturel-scw) [inline console/billing/how-to/add-payment-method.mdx:37]: replace 1€ by €1

File: console/billing/how-to/change-billing-information.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  - billing
+---
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/change-billing-information.mdx:22]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.

File: console/billing/how-to/change-payment-method.mdx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
+
+<Message type="requirement">
```
Comment (jpaturel-scw) [inline console/billing/how-to/change-payment-method.mdx:18]: Add here that a valid credit card is always required on the account, even if a sepa is added and setup as the default payment method.

File: console/billing/how-to/change-payment-method.mdx [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/change-payment-method.mdx:24]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.

File: console/billing/how-to/redeem-voucher-code.mdx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+A Scaleway voucher provides you with a discount or credit to use on our services. You can receive it as part of a promotional campaign or from your account manager. To redeem a voucher, you need to [create a Scaleway account](/console/account/how-to/create-an-account/).
+
+Voucher codes are unique and valid only for a defined period of time.
```
Comment (jpaturel-scw) [inline console/billing/how-to/redeem-voucher-code.mdx:18]: Add here that a voucher can only be used if the customer has already added a valid payment method.

File: console/billing/how-to/redeem-voucher-code.mdx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/billing/optimization) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/redeem-voucher-code.mdx:26]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.

File: console/billing/how-to/use-billing-alerts.mdx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```
Comment (jpaturel-scw) [inline console/billing/how-to/use-billing-alerts.mdx:33]: Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.

File: console/billing/how-to/use-billing-alerts.mdx [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+2. Click **Consumption**.
+3. Scroll down to find **Billing alerts**.
+4. Click **Create a billing alert** to configure an alert action. A pop-up displays.
+5. Enter the percentage threshold that triggers the alert and configure the warning that will be sent.
+    Available warning types are: 
+    * SMS notifications
+    * E-mail notifications
+    * Webhook triggers
+6. Click **Create billing alert** to enable the alert.
+    You may set up to 10 billing alerts. If you wish to create more, click **Add billing alert**, and repeat the steps above.
```
Comment (jpaturel-scw) [inline console/billing/how-to/use-billing-alerts.mdx:44]: @jsudraud How about specifying here that billing alerts can be set up on a single budget?

File: faq/billing.mdx [hunk 13]
```diff
@@ -98,8 +98,8 @@ The automatic payment of invoices occurs in the first few days of a month, once
 
 ## How can I manage my billing?
 
-You can manage your billing information from the [billing section](https://console.scaleway.com/organization/billing) of the Scaleway console. Your last invoices as well as the consumption report of the current month are available for download from there.
-Alternatively you can retrieve your consumption information and download your invoices using the [API](https://www.scaleway.com/en/developers/api/billing/).
+You can manage your billing from the [billing page](https://console.scaleway.com/billing/overview) of the Scaleway console. Your recent invoices as well as the consumption report of the current month are available for download.
```
Comment (jpaturel-scw) [inline faq/billing.mdx:101]: - replace link here by https://console.scaleway.com/organization/billing/payment

File: faq/dedibox-in-scaleway-console.mdx [hunk 14]
```diff
@@ -162,7 +162,7 @@ The definition of your default payment method follows these rules:
 * If you use different credit cards for Dedibox and Scaleway Cloud, the card registered with Scaleway Cloud becomes the default payment method for both Scaleway Cloud and Scaleway Dedibox invoices.
 
 <Message type="tip">
-  Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/organization/billing) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
+  Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/billing/payment) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
```
Comment (jpaturel-scw) [inline faq/dedibox-in-scaleway-console.mdx:165]: replace link here by https://console.scaleway.com/organization/billing/payment

**Review by jpaturel-scw — CHANGES_REQUESTED** [review]

File: bare-metal/dedibox/how-to/link-dedibox-account.mdx [hunk 15]
```diff
@@ -59,7 +59,7 @@ You can now manage your current and future Dedibox servers from the Scaleway con
...
         </Message>
     <Message type="note">
       Click **Choose antoher account** if you want to link your Dedibox account with a Scaleway account with a different email address.
```
Comment (pshambulingappa) [inline bare-metal/dedibox/how-to/link-dedibox-account.mdx:65]:
```suggestion
      Click **Choose another account** if you want to link your Dedibox account with a Scaleway account with a different email address. 
```

File: console/billing/how-to/use-billing-alerts.mdx (hunk 12 above)
Comment (jsudraud) [reply console/billing/how-to/use-billing-alerts.mdx:44]:
```suggestion
    You may set up to 10 billing alerts based on your monthly budget. To create a new billing alert, click **Add billing alert**, and repeat the steps above.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: console/billing/concepts.mdx [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+
+## Cost Manager
+
+Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, and costs. It offers a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.
+
+## Free Tier
+
+Scaleway offers free tiers for certain products. This means these products can be used at no cost, up to the specified usage limit of the free tier. These limits do not have an expiration date but may be subject to change. Eligible customers can enjoy these select services free of charge when creating an account, with a usage limit in place.
+
+## Free Trial
```
Comment (bene2k1) [inline console/billing/concepts.mdx:45]:
```suggestion
## Free trial
```

File: console/billing/concepts.mdx [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+
+## Consumption pie chart
+
+The consumption pie chart reflects the amount of money spent on the products you've used for the current month, excluding taxes and organized by product category. This chart also includes any applicable discounts. Its monetary value is calculated in real-time, providing an instant overview of your consumption.
+
+## Cost Manager
+
+Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, and costs. It offers a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.
+
+## Free Tier
```
Comment (bene2k1) [inline console/billing/concepts.mdx:41]:
```suggestion
## Free tier
```

File: console/billing/concepts.mdx [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+
+## Billing information
+
+Consists of information about the Organization that will be billed. They will appear on the invoice and can be modified on the Scaleway console. Find out more about [how to read an invoice](/en/docs/faq/billing/#how-to-read-my-invoice).
+
+## Consumption pie chart
+
+The consumption pie chart reflects the amount of money spent on the products you've used for the current month, excluding taxes and organized by product category. This chart also includes any applicable discounts. Its monetary value is calculated in real-time, providing an instant overview of your consumption.
+
+## Cost Manager
```
Comment (bene2k1) [inline console/billing/concepts.mdx:37]:
```suggestion
## Cost manager
```

File: faq/billing.mdx [hunk 19]
```diff
@@ -95,8 +79,8 @@ The status of an invoice can be:
 Prices announced on our website exclude all taxes.
 
 Because we are a European company, we are required to charge VAT on all orders made by European individual customers (B2C).
-The appropriate VAT rate is automatically applied to your invoice, depending on your fiscal residence. Just make sure your administrative details on your profile, postal address, recipient name, phone number, …) are in order.
-No other additional taxes, but European VAT is collected. 
+The appropriate VAT rate is automatically applied to your invoice, depending on your fiscal residence. Just make sure your administrative details on your profile, such as postal address, recipient name and phone number are in order.
```
Comment (nerda-codes) [inline faq/billing.mdx:82]:
```suggestion
The appropriate VAT rate is automatically applied to your invoice, depending on your fiscal residence. Just make sure your administrative details on your profile, such as postal address, recipient name, and phone number are in order.
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**